app/bundles/PluginBundle/Helper/IntegrationHelper.php

```diff
@@ -560,7 +559,7 @@ public function getShareButtons()
 
                 //add the api keys for use within the share buttons
                 $shareSettings['keys']   = $apiKeys;
-                $shareBtns[$integration] = $templating->render($plugin->getBundle().":Integration/$integration:share.html.twig", [
+                $shareBtns[$integration] = $this->twig->render($plugin->getBundle().":Integration/$integration:share.html.twig", [
```

Comment:
Note to self, we might find places where regex did not detect the change like `":Integration/$integration:share.html.twig"`